storage/jsonrpc.go

```diff
@@ -0,0 +1,34 @@
+package main
```

Comment:
Please add copyright to the `.go` files. Something like this to each file:

```
/*
 * Copyright (C) 2022 Dell Corporation
 * SPDX-License-Identifier: Apache-2.0
 */
```

Or whatever Dell is called.